automation/swift-components-docs/Sources/MozillaRustComponents/include/module.modulemap

```diff
@@ -0,0 +1,20 @@
+module MyCMap {
+    header "suggestFFI.h"
+    header "as_ohttp_clientFFI.h"
+    header "autofillFFI.h"
+    header "crashtestFFI.h"
+    header "errorFFI.h"
+    header "fxa_clientFFI.h"
+    header "loginsFFI.h"
+    header "nimbusFFI.h"
+    header "placesFFI.h"
+    header "pushFFI.h"
+    header "remote_settingsFFI.h"
+    header "rustlogforwarderFFI.h"
+    header "sync15FFI.h"
+    header "syncmanagerFFI.h"
+    header "tabsFFI.h"
```

Comment:
We really should create a way for UniFFI to help here.   Maybe if you use library mode and generate swift files, there could be an option for creating a single modulemap with everything in it.  In the meantime, I'm totally fine with hacking it with a hard-coded list of headers.

Reply:
I was "fine" with hardcoding it, because we currently have to hardcode any new component in the `generate-swift-project.sh` as well. But as soon as your PR lands with the library mode, we should auto-generate this `modulemap` file as well. I can put together a small script which does it as well as of now, so we don't hardcode anything here.